updatecli/values.yaml

```diff
@@ -4,5 +4,5 @@ github:
   username: "jenkins-infra-bot"
   token: "UPDATECLI_GITHUB_TOKEN"
   branch: "main"
-  owner: "jenkins-infra"
+  owner: "dduportal"
```

Comment:
What?? Here is the cause of https://github.com/jenkins-infra/docker-hashicorp-tools/pull/174 cc @dduportal